log

```diff
@@ -0,0 +1,311 @@
+============================= test session starts ==============================
```

Comment:
Thanks for putting an example here. I assume we should delete it before merging?